fedmsg_meta_fedora_infrastructure/compose2.py

```diff
@@ -80,5 +80,5 @@ def objects(self, msg, **config):
         elif 'file' in msg['msg']:
             return set([msg['msg']['file'].strip('/')])
         else:
-            compose = msg['msg']['compose_id']
+            compose = msg['msg'].get('compose_id', '')
```

Comment:
I believe #443 also contains this patch, except it leaves off the ```''``` and so defaults to ```None```. I also think testing it would be ideal, but LGTM otherwise.